src/LoginRouter.js

```diff
@@ -123,6 +123,7 @@ function (BaseLoginRouter,
       'signin/verify/u2f': 'verifyU2F',
       'signin/verify/generic_saml/assertion:saml2': 'verifySAMLFactor',
       'signin/verify/generic_oidc/assertion:oidc': 'verifyOIDCFactor',
+      'signin/verify/custom/claims_provider': 'verifyClaimsFactor',
```

Comment:
Will verify always have the provider? Asking because we are moving towards factors with only factorType.

Reply:
Good question. As we move towards factor profiles, the provider might become irrelevant. But that's the current model and we can't remove it right away.

Reply:
and we don't support multiple factors for claims yet, correct?

Reply:
@mauriciocastillosilva-okta That's right.